planning/sampling_based_planner/sampler_common/include/sampler_common/constraints/hard_constraint.hpp

```diff
@@ -17,10 +17,14 @@
 
 #include "sampler_common/structures.hpp"
 
+#include <vector>
 namespace sampler_common::constraints
 {
 /// @brief Check if the path satisfies the hard constraints
 MultiPoint2d checkHardConstraints(Path & path, const Constraints & constraints);
+bool has_collision(const MultiPoint2d & footprint, const MultiPolygon2d & obstacles);
+bool satisfyMinMax(const std::vector<double> & values, const double min, const double max);
```

Comment:
it looks like these functions are not defined anywhere.

Reply:
@maxime-clem They are defined here (already in universe): https://github.com/tier4/autoware.universe/blob/2117f81d9343e7384292ca660a319174e8b0bc63/planning/sampling_based_planner/sampler_common/src/sampler_common/constraints/hard_constraint.cpp#L25 
I just declared them on the header so they can be used by other modules.

Reply:
My bad :sweat: